tests/s3/test_object_tagging.py

```diff
@@ -627,38 +595,24 @@ def test_tagvalue256chars_2469(self):
     @CTFailOn(error_handler)
     def test_tag_value_512_unichars_2470(self):
         """Create a tag having values more than 512 Unicode characters in length."""
-        self.log.info(
-            "Create a tag having values more than 512 Unicode characters in length")
-        self.log.info(
-            "Creating a bucket with name %s",
-            self.bucket_name)
-        resp = self.s3_test_obj.create_bucket(
-            self.bucket_name)
+        self.log.info("Create a tag having values more than 512 Unicode characters in length")
+        self.log.info("Creating a bucket with name %s", self.bucket_name)
+        resp = self.s3_test_obj.create_bucket(self.bucket_name)
```

Comment:
Create bucket is used throughout the file. We can use common code in setup function or can make use of fixtures

Reply:
Using common code by @pytest.mark.usefixtures("create_bucket")